src/main/java/net/sf/jabref/gui/entryeditor/EntryEditor.java

```diff
@@ -962,6 +1000,16 @@ public void stateChanged(ChangeEvent event) {
                     ((EntryEditorTab) activeTab).updateAll();
                     activateVisible();
                 }
+
+                // When the tab "Related articles" gets selected, the request to get the recommendations is started.
+                if (((JTabbedPane) event.getSource()).getSelectedComponent().getName()
+                        .equals(Localization.lang("Related articles"))) {
+                    if (relatedAcrticlesTabNeedsRecommendations) {
```

Comment:
Is that possible? I would also like to see the code at places where it belongs. The `relatedArticlesTab` should do the recommendations requesting `onfocus` and not triggered here.